<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Excel CS v. 3.1416</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icono 3.png</normaloff>:/icono 3.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="Button_New">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>New</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/nuevo.png</normaloff>:/nuevo.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Button_Save">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/guardar.png</normaloff>:/guardar.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Button_Open">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/Abrir.png</normaloff>:/Abrir.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Button_Print">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Print</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/imprimir.png</normaloff>:/imprimir.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>395</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="actual_cell">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 255);
font: 75 10pt &quot;Kozuka Gothic Pro B&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="Formule_Bar"/>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="SpreadSheet">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Excel_CS_v_3_14_16"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/nuevo.png</normaloff>:/nuevo.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Abrir.png</normaloff>:/Abrir.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/guardar.png</normaloff>:/guardar.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imprimir.png</normaloff>:/imprimir.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/salir.png</normaloff>:/salir.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_Excel_CS_v_3_14_16">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icono 3.png</normaloff>:/icono 3.png</iconset>
   </property>
   <property name="text">
    <string>About Excel CS v 3.14.16</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
